lesson7_task1.py

```diff
@@ -0,0 +1,16 @@
+"""
+Make a program that has some sentence (a string) on input and returns a dict containing all unique words as keys
+and the number of occurrences as values.
+"""
+
+sentence = input("Enter a sentence: ").lower()
+words = sentence.split()
+
+unique_words = dict()
+
+for word in words:
```

Comment:
Nice solution, accepted, you could also try using defaultdict so that you don't need to check on key presence in dict.